clients/admin-ui/src/features/common/hooks/useFidesLang.tsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { find } from "lodash";
+import { ReactNode } from "react";
+
+import { useAppSelector } from "~/app/hooks";
+import {
+  selectDataSubjects,
+  useGetAllDataSubjectsQuery,
+} from "~/features/data-subjects/data-subject.slice";
+import {
+  selectDataUses,
+  useGetAllDataUsesQuery,
+} from "~/features/data-use/data-use.slice";
+import {
+  selectDataCategories,
+  useGetAllDataCategoriesQuery,
+} from "~/features/taxonomy";
+
+const useFidesLang = () => {
```

Comment:
I like the fideslang name, but later saw that it's generally referred to as Taxonomies. Should I rename it?

Reply:
I would personally associate the name Fideslang more with the dot notation strings than the display names; I think something like `useTaxonomy` would be good, yeah.

Reply:
Just renamed it to useTaxonomies, thank you!